.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,29 @@
-Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues)._
+Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues).
+For more information, see the [CONTRIBUTING Guide](https://github.com/Cloud-CV/Origami/blob/master/.github/CONTRIBUTING.md) 
```

Comment:
I was thinking of having something like this too https://github.com/angular/angular.js/blob/master/DEVELOPERS.md

Reply:
Are you referring to the contributing guide ?

Reply:
No, I was thinking that we should have a guide like the above for new developer to maintain a continuant style throughout origami. Take for instance https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-coding-rules

Reply:
We should make these changes in the contributing guide .

Reply:
Contributing guide would be fine, I just want this information to be present somewhere so as to enforce a continuant style throughout the project.